src/Styles/Hooks.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace WPML\PB\Cornerstone\Styles;
+
+use WPML\FP\Fns;
+use WPML\FP\Logic;
+use WPML\FP\Maybe;
+use WPML\LIB\WP\Post;
+
+class Hooks implements \IWPML_Backend_Action, \IWPML_Frontend_Action, \IWPML_DIC_Action {
+
+	const META_KEY = '_cs_generated_styles';
+
+	/** @var \WPML_PB_Last_Translation_Edit_Mode $lastEditMode */
+	private $lastEditMode;
+
+	/** @var \WPML_Cornerstone_Data_Settings $dataSettings */
+	private $dataSettings;
```

Comment:
@strategio You can now delete these two.

Reply:
Good catch, removed!